### ISSUE TITLE
Previous project's revisions to issues happening during an import of a new project 

### ISSUE THREAD

**Issue by jdewa** [solidify/jira-azuredevops-migrator#474]
hello, 

While importing a new project's issues into devops, the import is doing revisions and importing previous project's issues as well - is there any fix to this so that the runtime is quicker?

**Comment by Alexander-Hjelm** [comment]
Nope, not through config at least. You'll have to edit the source code
